FirebaseInAppMessaging/Swift/Source/SwiftUIPreviewHelpers.swift

```diff
@@ -0,0 +1,157 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import FirebaseInAppMessaging
+
+public struct InAppMessagingPreviewHelpers {
+  public static func cardMessage(campaignName: String = "Card message campaign",
+                                 title: String = "Title for modal message",
+                                 body: String? = "Body for modal message",
+                                 textColor: UIColor = UIColor.label,
+                                 backgroundColor: UIColor = UIColor.systemBackground,
+                                 portraitImage: UIImage = UIImage(systemName: "message")!,
+                                 landscapeImage: UIImage? = UIImage(systemName: "message.fill"),
```

Comment:
I wonder if we should use rectangles here: `rectangle.portrait` and `rectangle` (optionally adding `.fill` to either)

That way it shows the frame appropriately. WDYT?